src/StatusQ/Popups/StatusModal.qml

```diff
@@ -195,6 +195,12 @@ QC.Popup {
         \endqml
     */
     property bool hasFloatingButtons: false
+    /*!
+       \qmlproperty modalBackgroundColor
+        This property decides the modal background color
+        \endqml
+    */
+    property string modalBackgroundColor: Theme.palette.statusModal.backgroundColor
```

Comment:
Apart from the fact StatusModal is marked as deprecated, why not simply `backgroundColor`?

Reply:
You are right about the Modal being outdated. how about I do it under this task https://github.com/status-im/status-desktop/issues/6587. Will be touching the sendModal quite a bit. Have updated it in the task i just shared.
and I will rename it to background color 👍

Reply:
Sure :)